dts/bindings/adc/nxp,kinetis-adc16.yaml

```diff
@@ -20,8 +20,38 @@ properties:
     interrupts:
       required: true
 
+    dma-channel:
+      type: int
+      required: false
+      description: dma channel to use, for periodic trigger shall be 0-3
+
     "#io-channel-cells":
       const: 1
 
+    clk-source:
+      type: int
+      required: false
+      description: use alternate clock reference source
+
+    long-sample:
+      type: int
+      required: false
+      description: enable long sample mode
+
+    continuous-convert:
+      type: boolean
+      required: false
+      description: If use continuous convert
+
+    high-speed:
+      type: boolean
+      required: false
+      description: If use high speed
+
+    hw-trigger-src:
+      type: int
+      required: false
+      description: hardware trigger source
```

Comment:
Seems like a enum here.

Reply:
@galak  the source is defined as below, and different SOC has different trigger source defined, so I suppose kept as int type would be better.
 
![image](https://user-images.githubusercontent.com/800198/105576460-d9d0d780-5dad-11eb-965b-9e4991aee86a.png)

Reply:
Ok, maybe add a reference to the register/field in the UM one should look at for the values.

Reply:
good suggestion, let me add one comments in the driver code.

Reply:
Should add a comment in the binding as well.